README.md

```diff
@@ -97,3 +124,7 @@ To learn React, check out the [React documentation](https://reactjs.org/).
 
 [license-badge]: https://img.shields.io/github/license/fnndsc/chris_ui.svg
 [last-commit-badge]: https://img.shields.io/github/last-commit/fnndsc/chris_ui.svg
+[repo-link]: https://github.com/FNNDSC/ChRIS_ui
+[stars-badge]: https://img.shields.io/github/stars/FNNDSC/ChRIS_ui
```

Comment:
I love the `code-size` badge, though as my last nitpicks, I think `stars-badge` and `forks-badge` are superfluous. It's already displayed on Github.